.dockerignore

```diff
@@ -14,7 +14,10 @@ src/types
 project.yaml
```

Comment:
In the future, when we have dependent PRs, please do the following

1. In the title, say: `[DO NOT MERGE UNTIL UPDATING BASE]...`
2. Make the base PR the other branch, not `main`

RIght now, I believe it reflects all the changes in the other PR as well.

Reply:
1. Ok
2. If I set other branch as base, so far I known, I can't change it later.

Reply:
Re 2:

https://github.com/user-attachments/assets/aedcb99c-c479-4f43-81ba-4a6c31cc2a49

Reply:
Updated